git-coverage

```diff
@@ -690,6 +691,22 @@ def getopt_keep_dashes(argv, short, long):
     args += after_dashes
     return opts, args
 
+def read_coveragerc(coveragerc, key):
```

Comment:
If you're passing in a key you should also pass in the section rather than hardcode it.